allennlp/data/tokenizers/word_stemmer.py

```diff
@@ -26,7 +30,6 @@ def from_params(params: Params) -> 'WordStemmer':
         params.assert_empty('WordStemmer')
         return word_stemmers[choice]()
 
-
```

Comment:
We want this blank line - python style is two blank lines between top-level classes, right?